src/pages/profile/Profile.tsx

```diff
@@ -51,6 +57,20 @@ const Profile: FC<ProfileProps> = ({ user }) => {
           <DataLine title={'Ник'} value={user.display_name}></DataLine>
           <DataLine title={'Телефон'} value={user.phone}></DataLine>
         </InfoSt>
+        <ButtonSettings onClick={toggleVisible}>
+          {'Изменить данные'}
```

Comment:
Можно не менять, но вообще для строк не обязательно добавлять {}

```suggestion
          Изменить данные
```